### ISSUE TITLE
Fix `TripleDES` issue that's blocking `cryptography` dependency update 

### ISSUE THREAD

**Issue by rdimaio** [rucio/rucio#7075]
### Description

https://github.com/rucio/rucio/pull/7073

Errors:
```
E   assert not '/opt/venv/lib64/python3.9/site-packages/paramiko/pkey.py:100: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.\n  "cipher": algorithms.TripleDES,\n/opt/venv/lib64/python3.9/site-packages/paramiko/transport.py:259: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.\n  "class": algorithms.TripleDES,\n'
```

### Steps to reproduce

N/A

**Comment by rdimaio** [comment]
Related: https://github.com/paramiko/paramiko/issues/2419

**Comment by rdimaio** [comment]
This was fixed in `paramiko==3.4.1`: https://www.paramiko.org/changelog.html

**Comment by bari12** [comment]
Backport 35.6.2
